frontend/src/pages/notebookController/screens/server/BrowserTabPreferenceCheckbox.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { Checkbox, FormGroup } from '@patternfly/react-core';
+
+type BrowserTabPreferenceCheckboxProps = {
+  isChecked: boolean;
+  setChecked: (checked: string) => void;
+};
```

Comment:
Maybe this component is unique -- it doesn't need to interact with the form -- we should probably just include all state about it including where it is pulling from inside this component.